public/app/features/datasources/components/picker/DataSourcePicker.test.tsx

```diff
@@ -281,6 +294,8 @@ describe('DataSourcePicker', () => {
       await user.keyboard('foobarbaz'); //Search for a DS that should not exist
 
       expect(await screen.findByText('Configure a new data source')).toBeInTheDocument();
+      // It should point to the new data source page including any sub url configured
+      expect(screen.getByRole('link')).toHaveAttribute('href', '/my-sub-url/datasources/new');
```

Comment:
@ivanortegaalba  should be `'/my-sub-path/connections/datasources/new'` instead of `'/my-sub-path/datasources/new'` i think

Reply:
Fixed! Good catch!